app/controllers/actions/AuthAction.scala

```diff
@@ -106,4 +109,21 @@ class AuthActionImpl @Inject()(override val authConnector: AuthConnector,
             Future.successful(Redirect(controllers.error.routes.ErrorController.unauthorised()))
         }
     }
+
+  private def checkIfUserErnCanAccessCoD[A](ernFromUrl: String,
+                                            internalId: String,
+                                            credId: String,
+                                            sessionId: Option[SessionId],
+                                            hasMultipleEnrolments: Boolean
+                                           )(block: UserRequest[A] => Future[Result])
+                                           (implicit request: Request[A]): Future[Result] = {
+    val userType = UserType(ernFromUrl)
+
+    if (Seq(Unknown, GreatBritainWarehouse, NorthernIrelandWarehouse).contains(userType)) {
+      logger.warn(s"[checkIfUserErnCanAccessCaM] User attempted to access CaM with invalid ern: '$ernFromUrl'")
```

Comment:
```suggestion
      logger.warn(s"[checkIfUserErnCanAccessCOD] User attempted to access COD with invalid ern: '$ernFromUrl'")
```

Reply:
Addressed in https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/57/commits/9bbd4de2d48fe7682f5b386720efe2dd1575e9cf